packages/app/src/app/overmind/effects/vscode/ModelsHandler.ts

```diff
@@ -185,6 +185,19 @@ export class ModelsHandler {
     });
   }
 
+  public async setModuleCode(module: Module) {
+    const moduleModel = this.getModuleModel(module);
+    const model = await moduleModel.model;
+
+    if (!model) {
+      return;
+    }
+
+    this.isApplyingOperation = true;
```

Comment:
We might need to make this an option in the future, I can think of some use cases where we want to set the code and send the changes. But not needed for now!